code/modules/martial_arts/combos/judo/armbar.dm

```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/judo/armbar
+	name = "Armbar"
+	steps = list(MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_DISARM,MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "place an opponent on the ground into an armbar, immobilizing them"
+	combo_text_override = "Grab, swap hands, disarm, grab, swap hands, harm"
+
+/datum/martial_combo/judo/armbar/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(target.IsWeakened() || !IS_HORIZONTAL(target))
```

Comment:
Why are you checking for weakened here?

Reply:
All CQC combos check for weaken, so I wasn't sure if it was vital to the functioning of combos.

Reply:
Update: it is not, and the check has been removed.